components/search/MoreFacetsDropdown.vue

```diff
@@ -45,7 +56,28 @@
         type: Array,
         default: () => []
       }
+    },
+
+    data() {
+      return {
+        selected: {}
+      };
+    },
+
+    computed: {
+      selectedAmount() {
+        let sum = 0;
+        for (const key in this.selected) {
+          sum += this.selected[key].length;
+        }
+        return sum;
+      }
+    },
+
+    methods: {
+      updateSelected(value) {
```

Comment:
This isn't so easy to scan.

Let's rewrite it as:
```js
      updateSelected(facetName, selectedFields) {
        Vue.set(this.selected, facetName, selectedFields);
      }
```
... with the accompanying change to `selected`, below.

Reply:
Yeah much cleaner.